python/sonata/__init__.py.in

```diff
@@ -1,3 +1,3 @@
-from ._sonata import *
```

Comment:
I meant that the folder in which this file exists should be renamed, no ?

Reply:
OK, thanks.

It did not affect the build, but I have renamed the source folder nevertheless for the sake of consistency.